multimedia/x265/Portfile

```
@@ -30,9 +30,8 @@ long_description    x265 is a free software library and application for \
                     compression format, and is released under the terms of the \
                     GNU GPL.
 homepage            https://www.videolan.org/developers/x265.html
-master_sites        https://download.videolan.org/pub/videolan/x265/
-distname            ${name}_${version}
-worksrcdir          ${distname}/source
+github.tarball_from archive
+worksrcdir          ${name}-${version}/source
```

I would expect you not to need to change this, though I haven't tested it.
```suggestion
worksrcdir          ${distname}/source
```

---

I used this because the tarball is called 3.4.tar.gz and I want to name the directory x265-3.4